plutus-tx/src/PlutusTx/Base.hs

```diff
@@ -0,0 +1,58 @@
+module PlutusTx.Base (fst, snd, curry, uncurry, ($), flip, until, (.), const, id) where
+
+import           PlutusTx.Bool
+
+{-# INLINABLE fst #-}
```

Comment:
I assume these are all functions where we can't actually use the base versions?

Reply:
Not actually. Some of them were definied in the `Functor` module, and some were imported from `Prelude`. I decided to merge them into one place and define Plutus versions.